AUTHORS.md

```diff
@@ -38,3 +38,4 @@ All names are sorted alphabetically by last name. Contributors, please add your
 - [Vlad Iliescu](https://vladiliescu.net)
 - [Hrittik RoyChoudhury](https://github.com/Hrittik20)
 - [Tamara Atanasoska](https://github.com/tamaraatanasoska)
+- [Tahar Allouche](https://github.com/taharallouche)
```

Comment:
Should this file be removed? Probably way outdated by now...

Reply:
Tbh I never questioned its existence, I added my name I believe super late, a month-two ago. I thought of it as a historical record of who had contributed in the past. Do you think we don't need it?

Reply:
If so we can also remove it as part of this PR, that would be ok.

Reply:
Removed it in https://github.com/fairlearn/fairlearn/pull/1483/commits/676ad6a53e17d3c39b45c21f0c679b2cd3e8499f